hoot-core/src/main/cpp/hoot/core/scoring/MatchComparator.cpp

```diff
@@ -646,59 +642,43 @@ void MatchComparator::_tagTestOutcome(const OsmMapPtr& map, const QString uuid,
                                       const QString expected, const QString actual)
 {
   SetTagVisitor stv1(MetadataTags::HootExpected(), expected);
-  MatchComparator::UuidToEid::iterator it = _actualUuidToEid.begin();
-  while (it != _actualUuidToEid.end())
-  {
-    if (it.key().contains(uuid))
-    {
-     boost::shared_ptr<Element> eid = map->getElement(it.value());
-      stv1.visit(eid);
-    }
-    it++;
-  }
-
```

Comment:
Two almost identical loops one to call `stv1.visit(eid)` and the other to call `stv2.visit(eid)` so they can be consolidated into one and use a for loop with the prefix `++` operator.